stable/airflow/values.yaml

```diff
@@ -4,6 +4,35 @@
 ##
 ## common settings and setting for the webserver
 airflow:
+
+  ##
+  ## secretsMapping can be overridden in values.yaml as such:
+  ## secretsMapping:
+  ## - envVar: AIRFLOW__LDAP__BIND_PASSWORD
+  ##   secretName: ldapBindPassword
+  ## - envVar: AIRFLOW__ATLAS__PASSWORD
+  ##   secretName: atlasPassword
+  ## - envVar: AIRFLOW__SMTP__PASSWORD
+  ##   secretName: smtpPassword
+  ## - envVar: AIRFLOW__KUBERNETES__GIT_PASSWORD
+  ##   secretName: kubernetesGitPassword
+  ## - envVar: POSTGRES_USER
+  ##   secretName: postgresUser
+  ## - envVar: POSTGRES_PASSWORD
+  ##   secretName: postgresPassword
+  ## - envVar: REDIS_PASSWORD
+  ##   secretName: redisPassword
+  secretsMapping:
+
+  ## used only when existingAirflowSecrets is false
+  defaultSecretsMapping:
```

Comment:
will this be still compatible with "airflow.secret" template users?

and secretName look like it refers to the key inside your single secret, so maybe "secretKey" or "secretDataName"

Reply:
It's backwards compatible in the new change. If they had provided and existingAirflowSecret with the postgresPassword etc. it will now work out of the box.